index.html

```diff
@@ -34,7 +34,7 @@ <h1>Raspberry Pi Pico Pinout</h1>
         </noscript>
     </nav>
     <article class="pinout" id="pinout">
-        <table aria-label="Lefthand pins" class="labels left" aria-rowcount="20" aria-colcount="5" cellpadding="5" cellspacing="5">
+        <table aria-label="Lefthand pins" class="labels left" aria-rowcount="20" aria-colcount="5" cellpadding="0" cellspacing="5">
```

Comment:
I know very little about webdev, but "should" this be in the CSS file, rather than hardcoded in the HTML, so that you'd only need to fix it once (in the CSS) rather than individually in each of the HTML files? :man_shrugging:

Reply:
This is explicitly for the very rare and odd occasions where there's no CSS being used at all. Try it :laughing:

Reply:
Ahhhh, fair enough. I know so little about webdev that I didn't even know supporting non-CSS browsers was a thing :laughing: